src/deployments/kubernetes/README.md

```diff
@@ -12,21 +12,23 @@ For local development, you need to use
   environment.
 * Set the docker environment inside minikube:
 
-`eval $(minikube docker-env)` for Unix shells or
-`minikube docker-env | Invoke-Expression` in PowerShell
+`eval $(minikube docker-env)` for Unix shells or `minikube docker-env |
+Invoke-Expression` in PowerShell
 
-* From the location of the [Dockerfile](src\Dockerfile), Build the data management service image:
-`docker build -t local/edfi-data-management-service:latest .`
+* From the location of the [Dockerfile](src\Dockerfile), Build the data
+management service image: `docker build -t
+local/edfi-data-management-service:latest .`
 
-> [!WARNING]
-> The [deployment file](./data-management-service-deployment.yaml#L21) has a policy
-> of Never pull, which means that will use the locally built image instead of trying
-> to pull the image from Docker Hub which is the default behavior, remove
-> once the image is published to the registry.
+> [!WARNING] The [deployment
+> file](./data-management-service-deployment.yaml#L21) has a policy of Never
+> pull, which means that will use the locally built image instead of trying to
+> pull the image from Docker Hub which is the default behavior, remove once the
+> image is published to the registry.
```

Comment:
```suggestion
> [!WARNING]
> The [deployment file](./data-management-service-deployment.yaml#L21) has a
> policy of Never pull, which means that will use the locally built image instead of
> trying to pull the image from Docker Hub which is the default behavior, remove
> once the image is published to the registry.
```